metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -171,10 +193,14 @@ def create(
         max_workers,
         workflow_timeout,
         workflow_priority,
+        ignore_events,
     )
 
     if only_json:
         obj.echo_always(str(flow), err=False, no_bold=True)
+        if flow.trigger_on_template() is not None:
```

Comment:
We can skip printing the sensor template or introduce a CLI arg that prints just that.